metricflow-semantics/metricflow_semantics/query/query_resolver.py

```diff
@@ -321,7 +321,7 @@ def _resolve_order_by(
             else:
                 order_by_specs.append(
                     OrderBySpec(
-                        instance_spec=matching_specs[0],
+                        instance_spec=matching_specs[0].with_alias(None),
```

Comment:
same here about needing to call `with_alias`

Reply:
This one we'll keep - adding a comment as to why:
`# Ignore aliases in the order by since we'll render the expression instead of the alias.`